Assets/StreamingAssets/Tables/QuestList-Classic.txt

```diff
@@ -237,7 +237,7 @@ R0C20Y07, Nobility, N, 20,
 R0C20Y22, Nobility, N, 20,
 R0C30Y25, Nobility, N, 30,
 -R0C40Y23, Nobility, N, 40, BROKEN. QRC file is missing in DF+DFQFIX
-R0C4XY23, Nobility, N, 4X,
+-R0C4XY23, Nobility, N, 4X, Needs support for until _thing_ performed:
```

Comment:
This is already a supported task type (see line 235 of Task.cs). I'll add this quest to my internal tracker to take a look at later, as it doesn't sound like this quest is executing properly.

Reply:
Ah, silly me, I simply checked the action classes and found none called until. Can't recall what led me to look at it now.